packages/earth-map/src/components/org-switcher/component.tsx

```diff
@@ -1,11 +1,14 @@
 import React, { useContext, useState } from 'react';
-import { Auth0Context } from 'auth/auth0';
+import { remove } from 'lodash';
+import classNames from 'classnames';
```

Comment:
don't think logo should be in org-switcher, either rename the component in something more generic (eg header) or extract if from here

Reply:
Will rename it to <Header/>